deployment/src/main/java/io/quarkus/neo4j/deployment/DevServicesBuildTimeConfig.java

```diff
@@ -22,7 +22,7 @@ public class DevServicesBuildTimeConfig {
     /**
      * The container image name to use, for container based DevServices providers.
      */
-    @ConfigItem(defaultValue = "neo4j:4.4")
+    @ConfigItem(defaultValue = "neo4j:5.19.0")
```

Comment:
```suggestion
    @ConfigItem(defaultValue = "neo4j:5")
```

We used to use the 4.4 latest, and I would like to keep the semantics, with 5 having monthly minors. We (Neo4j) stopped doing patches on the product except on emergencies.